Maple2.Server.Game/Manager/Items/InventoryManager.cs

```diff
@@ -12,6 +12,7 @@
 using Maple2.Server.Game.Packets;
 using Maple2.Server.Game.Session;
 using Maple2.Tools.Extensions;
+using Microsoft.Scripting.Utils;
```

Comment:
remove

Reply:
This is needed to add range on line 281

Reply:
There's no precedence to use this dependency, just iterate to add:

```cs
for (Item item in session.Item.Inventory.Find(ingredient.ItemId, ingredient.Rarity)) {
    value.Add(item);
}
```